coala_quickstart/info_extractors/PackageJSONInfoExtractor.py

```diff
@@ -0,0 +1,45 @@
+import json
+
+from coala_quickstart.info_extraction.InfoExtractor import InfoExtractor
+from coala_quickstart.info_extraction.Information import (
+    LicenseUsedInfo, ProjectDependencyInfo, IncludePathsInfo, ManPathsInfo,
+    VersionInfo)
+
+
+class PackageJSONInfoExtractor(InfoExtractor):
+    supported_file_globs = ("package.json",)
```

Comment:
`.` matches only `.`, just verified :)
`packagexjson` fails for glob "package.json"
and `package.json` falis for escaped glob "package/.json"